.circleci/config.yml

```
@@ -0,0 +1,65 @@
+var_1: &docker_image circleci/node:10.13.0-browsers
+var_2: &cache_key raiden-webui-{{ .Branch }}-{{ checksum "package-lock.json" }}
+
+
+anchor_1: &defaults
```

anchor_1 looks much like an executor 
for every job you can define an executor meaning like waht are the propertys of the underlying machine/image you wish to run your job in

declaring your executors would look smth like this: 

```suggestion
executors:
  **ex-name**:
    working_directory: ~/src
    docker:
      - image: *docker_image
```

You can also define resources the underlying machine should have and environment variables